modules/oadp-self-service-namespace-permissions.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// backup_and_restore/application_backup_and_restore/oadp-self-service/oadp-self-service.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-self-service-namespace-permissions_{context}"]
+= {oadp-short} Self-Service namespace permissions
+
+As a cluster administrator, ensure that a namespace admin user has editor roles assigned for the following list of objects in their namespace. These objects ensure that a namespace admin user can perform the backup and restore operations in their namespace.  
+
+* `nonadminbackups.oadp.openshift.io`
+* `nonadminbackupstoragelocations.oadp.openshift.io`
+* `nonadminrestores.oadp.openshift.io`
+* `nonadmindownloadrequests.oadp.openshift.io`
+
+A namespace admin user cannot edit the `NonAdminBackupStorageLocation` (NABSL) CR. They can only create a new NABSL.
+
+[id="oadp-self-service-yaml-backup-operation_{context}"]
+== Example `RBAC` YAML for backup operation
+
+Following is an example of the `RBAC` YAML file with namespace permissions for a namespace admin user to perform a backup operation.
+
+.Example `RBAC`
+[source,yaml]
+----
+...
+- apiGroups:
+      - oadp.openshift.io
+    resources:
+      - nonadminbackups
+      - nonadminrestores
+      - nonadminbackupstoragelocations
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - oadp.openshift.io
+    resources:
+      - nonadminbackups/status
```

Comment:
@shdeshpa07 Maybe we could add this line and remove the restore permission section
  
    - nonadminrestores/status